brave/brave-core
# Updated sidebar settings button target url

fix https://github.com/brave/brave-browser/issues/21060

Introduced new IDS_SETTINGS_SIDEBAR_SHOW_OPTION_TITLE.
Although it uses same string with IDS_SIDEBAR_SHOW_OPTION_TITLE,
made different string id to prevent future regression.
IDS_SIDEBAR_SHOW_OPTION_TITLE was shared by many other UI such as app menu
or context menu.


Resolves 

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Click sidebar settings button at the bottom of sidebar UI
2. Check `brave://settings/?search=Show%20Sidebar` is loaded at current tab
